language/e2e-tests/src/tests/write_set.rs

```diff
@@ -75,7 +75,12 @@ fn verify_and_execute_writeset() {
         1,
     );
     let output = executor.execute_transaction(writeset_txn.clone());
-    let expected_err = VMStatus::new(StatusCode::ABORTED).with_sub_status(11);
+    let expected_err = VMStatus::new(StatusCode::ABORTED)
```

Comment:
this is going to be the most brittle test ever.
But I like that we have one case that checks we print the function/offset.
Do you think this is too much? you want more?

Reply:
Took the words out of my mouth...I think this is going to be super brittle.

What about creating a functional test that the only thing it does is aborts and check this there? That should be less brittle since it's the only instruction and less subject to compilation deltas.

Reply:
Like Tim's idea; seems a bit funky to have the test for this behavior under `tests/write_set.rs`...

Reply:
I see there is no appreciation for slackers like me... insulting...